mne/preprocessing/tests/test_scalp_coupling_index.py

```diff
@@ -0,0 +1,64 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#          Alexandre Gramfort <alexandre.gramfort@inria.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+
+import pytest
+import numpy as np
+
+from mne.datasets.testing import data_path
+from mne.io import read_raw_nirx
+from mne.preprocessing.nirs import optical_density, scalp_coupling_index
+from mne.datasets import testing
+
+fname_nirx_15_0 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_0_recording')
+fname_nirx_15_2 = op.join(data_path(download=False),
+                          'NIRx', 'nirx_15_2_recording')
+fname_nirx_15_2_short = op.join(data_path(download=False),
+                                'NIRx', 'nirx_15_2_recording_w_short')
+
+
+@testing.requires_testing_data
+@pytest.mark.parametrize('fname', ([fname_nirx_15_2_short, fname_nirx_15_2,
+                                    fname_nirx_15_0]))
+@pytest.mark.parametrize('fmt', ('nirx', 'fif'))
+def test_scalp_coupling_index(fname, fmt, tmpdir):
+    """Test converting NIRX files."""
+    assert fmt in ('nirx', 'fif')
+    raw = read_raw_nirx(fname)
+    raw = optical_density(raw)
+    sci = scalp_coupling_index(raw)
+
+    # All values should be between -1 and +1
+    assert (sci < 1.0).all()
+    assert (sci > -1.0).all()
+
+    # Fill in some data with known correlation values
+    new_data = np.random.rand(raw._data[0].shape[0])
+    # Set first two channels to perfect correlation
+    raw._data[0] = new_data
+    raw._data[1] = new_data
+    # Set next two channels to perfect correlation
+    raw._data[2] = new_data
+    raw._data[3] = new_data * 0.3  # check scale invariance
+    # Set next two channels to anti correlation
+    raw._data[4] = new_data
+    raw._data[5] = new_data * -1.0
+    # Set next two channels to be uncorrelated
+    # TODO: this might be a bad idea as sometimes random noise might correlate
+    raw._data[6] = new_data
+    raw._data[7] = np.random.rand(raw._data[0].shape[0])
+    # Check values
+    sci = scalp_coupling_index(raw)
+    np.testing.assert_almost_equal(sci[0], 1)
```

Comment:
`from numpy.testing import assert_allclose` at the top then just do `assert_allclose` here. It's more flexible and has nicer options rtol/atol

Reply:
Thanks. Done